src/Model/PrimaryTypeGo.cs

```diff
@@ -136,5 +137,55 @@ public virtual string ImplementationName
             builder.Append('"');
             return builder.ToString();
         }
+
+        /// <summary>
+        /// Gets the expression for a zero-initialized primary type.
+        /// </summary>
+        public string ZeroInitExpression
+        {
+            get
+            {
+                switch (KnownPrimaryType)
+                {
+                    case KnownPrimaryType.ByteArray:
+                        return "nil";
+
+                    case KnownPrimaryType.Boolean:
+                        return "false";
+
+                    case KnownPrimaryType.Date:
+                        return "date.Date{}";
+
+                    case KnownPrimaryType.DateTime:
+                        return "date.Time{}";
+
+                    case KnownPrimaryType.DateTimeRfc1123:
+                        return "date.TimeRFC1123{}";
+
+                    case KnownPrimaryType.Double:
+                    case KnownPrimaryType.Decimal:
+                    case KnownPrimaryType.Int:
+                    case KnownPrimaryType.Long:
+                        return "0";
+
+                    case KnownPrimaryType.Base64Url:
+                    case KnownPrimaryType.String:
+                        return "\"\"";
+
+                    case KnownPrimaryType.Object:
+                        return "map[string]interface{}{}";

```

Comment:
> return "map[string]interface{}{}"; [](start = 23, length = 35)

I am sure we have been generating something not as pretty for objects.

Reply:
Do you know what we are generating?

Reply:
`*map[string]*map[string]interface{}`

Reply:
Thanks will fix.

Reply:
I dug into this more, object is map[string]interface{}, map[string]map[string]interface{} is an object with sub-objects.  So I will leave this as-is.